build.gradle

```diff
@@ -53,19 +53,29 @@ dependencies {
     annotationProcessor group: "io.kestra", name: "processor", version: kestraVersion
     compileOnly group: "io.kestra", name: "core", version: kestraVersion
 
+    // Langchain4j model providers
     implementation "dev.langchain4j:langchain4j"
     implementation "dev.langchain4j:langchain4j-anthropic"
     implementation "dev.langchain4j:langchain4j-azure-open-ai"
     implementation "dev.langchain4j:langchain4j-bedrock"
-    implementation "dev.langchain4j:langchain4j-elasticsearch"
     implementation "dev.langchain4j:langchain4j-google-ai-gemini"
     implementation "dev.langchain4j:langchain4j-mistral-ai"
     implementation "dev.langchain4j:langchain4j-ollama"
     implementation "dev.langchain4j:langchain4j-open-ai"
-    implementation "dev.langchain4j:langchain4j-pgvector"
     implementation "dev.langchain4j:langchain4j-vertex-ai"
     implementation "dev.langchain4j:langchain4j-vertex-ai-gemini"
 
+    // Langchain4j embeddings stores
+    implementation "dev.langchain4j:langchain4j-chroma"
+    implementation "dev.langchain4j:langchain4j-elasticsearch"
+    implementation "dev.langchain4j:langchain4j-embeddings-all-minilm-l6-v2"
```

Comment:
What is it used for? It didn't seems like an embedding store on its own?

Reply:
It seems to be used only for test, if so, add it as test dependency.

Reply:
Yes it's for test purposes so `testImplementation` you're right!